src/utils/validation.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * 사용자의 입력값이 유효한지 확인합니다.
+ * @param {string} numString - 입력 받은 문자열
+ * @param {number} length - 입력 받아야 하는 문자열의 길이
+ * return 값은 없으며, 유효하지 않을 경우 throw 문을 통해 예외를 발생시키고 프로그램을 종료합니다.
+ */
+export function isInputValid(numString, length) {
+  const INPUT_NUMS = numString.split("");
+  if (INPUT_NUMS.length !== length) {
+    throw new Error(`[ERROR] 입력한 값이 ${length} 자리가 아닙니다.`);
+  }
+  for (let num of INPUT_NUMS) {
+    if (isNaN(num)) {
+      throw new Error("[ERROR] 숫자만 입력할 수 있습니다.");
+    }
+  }
```

Comment:
```suggestion
    if (isNaN(numString)) {
      throw new Error("[ERROR] 숫자만 입력할 수 있습니다.");
    }
```
여기를 배열로 바꿔서 순회하시면서 개별로 처리해주셨는데 바로 입력받은 문자열에 `isNaN()` 하시는건 어떻게 생각하시나요??

Reply:
`numString`은 문자열이기 때문에 `isNaN(numString)`은 항상 `true`가 되지 않을까요?
입력받은 문자열 안의 각 문자들이 모두 숫자여야해서 반복문으로 일일이 확인해주었습니다!

Reply:
제가 알기론 `isNaN()`은 인자로 받은 값을 Number 타입으로 변환해서 `NaN`인지 체크하는거라서 문제 없을 것 같아요!
대신 내부적으로 공백이랑 이런게 체크가 안되기는 하는데 저희는 입력 값 받을 때 유효성 검사를 거쳐서 이 문제는 괜찮을 것 같네요🙃
그래도 유효성 검사가 완벽히 통과되었다고 보장하기는 어려우니깐 지예님 방식도 좋은 것 같습니다!
![스크린샷 2024-09-20 오전 10 20 04](https://github.com/user-attachments/assets/eaa32fdc-7d32-4492-a766-f3456b138951)

Reply:
와.. 몰랐어요ㅠ 하나 또 배워갑니다! 정말 감사해요 :+1: :+1: :+1: 이런 단순한 것도 모르고 있었네요 저는..ㅎㅎ